tokio/src/lib.rs

```diff
@@ -100,9 +105,11 @@ pub mod net;
 pub mod prelude;
 
 #[cfg(feature = "process")]
+#[cfg(not(loom))]
```

Comment:
Why? And should this use `cfg(all())` ?

Reply:
Tests don't work when running loom tests. I opted to disable the entire mod to speed up compilation time.

I am unsure if `all` is easier to read vs. two lines. I can switch though.

Reply:
We use `all` elsewhere, so we should probably stick to one or the other. I don't care which one.

I think I'd prefer to just disable the loom _tests_ for that module if possible, but don't consider this a blocker.

Reply:
Done